pygazebo/pygazebo.cc

```diff
@@ -252,18 +253,37 @@ class Agent : public Model {
             reinterpret_cast<const uint8_t*>(sensor->ImageData())));
   }
 
-  bool TakeAction(const std::map<std::string, double>& forces) {
+  bool TakeAction(const std::map<std::string, double>& controls,
```

Comment:
As you suggested, we may add position control later. So we may use position target to control some joint, while using velocity to control some other joints. If we want to allow that possibility, we should separate different arguments for these different control mode: force, velocity, and position